docs/io/output/rpacket_tracking.ipynb

```diff
@@ -43,12 +43,7 @@
    "id": "4b0de6ca",
```

Comment:
<p>Use normal parantheses <code>()</code> instead of curly ones (same above) </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tardis-sn/tardis/pull/2073/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='46'/>

Reply:
Actually, the conventions (like using bullet points, curly braces, etc. that you have mentioned) I have used in the documentation are the same ones that were used earlier (i.e. the above part of documentation which was not written by me). So, I followed that same documentation format. But I have made changes in the entire rpacket_tracking documentation using the conventions you told so that it looks in a uniform format. Hope it's fine.